corehq/apps/domain/views/accounting.py

```diff
@@ -963,13 +963,9 @@ def is_form_editable(self):
 )
 
 
-class SelectPlanView(DomainAccountingSettings):
-    template_name = 'domain/select_plan.html'
-    urlname = 'domain_select_plan'
+class PlanViewBase(DomainAccountingSettings):
```

Comment:
I don't know if we really use "Base" view classes like this, but I couldn't think of another good way to do this without wading through every piece of logic in the existing view.